# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/Scope.h
# Commit: 2a0aacbd11de
# Full Hash: 2a0aacbd11dee6907445c902aa088ab4b9492d6b
# Author: caroline <cullen.caroline@gmail.com>
# Date: 2019-12-10 21:29:05
# Regressor Bug: 1588861
# File Overlap Count: 1
# Description:
#   Bug 1588861 - Part 3: Add support for xdr encoding modules into xdr cache. r=iain,tcampbell
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D54158
# ==============================================================================

diff -r 639290e9ff22 -r 2a0aacbd11de js/src/vm/Scope.h
--- a/js/src/vm/Scope.h	Tue Dec 10 02:52:30 2019 +0000
+++ b/js/src/vm/Scope.h	Tue Dec 10 02:52:37 2019 +0000
@@ -919,6 +919,10 @@
                              Handle<ModuleObject*> module,
                              HandleScope enclosing);
 
+  template <XDRMode mode>
+  static XDRResult XDR(XDRState<mode>* xdr, HandleModuleObject module,
+                       HandleScope enclosing, MutableHandleScope scope);
+
  private:
   static ModuleScope* createWithData(JSContext* cx,
                                      MutableHandle<UniquePtr<Data>> data,